galgebra/mv.py

```diff
@@ -508,30 +512,24 @@ def __mul__(self, A):
 
             selfxA = Mv(self.Ga.mul(self.obj, A.obj), ga=self.Ga)
             selfxA.is_blade_rep = False
-            selfxA = selfxA.blade_rep()
+            return selfxA.blade_rep()
 
-            self = self.blade_rep()
-            A = A.blade_rep()
```

Comment:
Now that these don't mutate state, there is no need to try and undo the swap between representations

Reply:
There are other places in this file that simply did not swap between representations